aidial_sdk/deployment/from_request_mixin.py

```diff
@@ -35,6 +36,11 @@ class FromRequestDeploymentMixin(FromRequestMixin):
     api_version: Optional[StrictStr] = None
     headers: Mapping[StrictStr, StrictStr]
 
+    original_request: fastapi.Request = Field(..., exclude=True)
```

Comment:
Why do we need "..." here? Does the behavior really change if you don't specify three dots?

Reply:
It appears to be useful only for documentation according to the [Pydantic docs](https://docs.pydantic.dev/latest/concepts/models/#required-fields).

> You may also use Ellipsis/... to emphasize that a field is required, especially when using the Field constructor.

Let's leave it.